### PR TITLE
report changes modified

### PR DESCRIPTION
**Pull request by aman-tunerlabs**

### CODE REVIEW

File: module/solutions/helper.js [hunk 1]
```diff
@@ -127,4 +127,24 @@ module.exports = class solutionsHelper {
 
     }
 
+    static generateSolutionsExternalId(solutionId){
```
Comment (ajithtunerlabs) [inline module/solutions/helper.js]: remove this one and use solutionDocument function in solutionHelper

File: controllers/v1/reportsController.js [hunk 2]
```diff
@@ -1287,17 +1269,10 @@ module.exports = class Reports {
...
-            status: 400,
-            message: "From Date cannot be greater than to date !!!"
-          })
-        }
         
         let solutionDocument = await database.models.solutions.findOne({
-          programExternalId:req.params._id,
-          externalId:req.query.solutionExternalId     
+          _id:req.params._id,
+          // externalId:req.query.solutionExternalId
```
Comment (ajithtunerlabs) [inline controllers/v1/reportsController.js]: please remove commented code

**Comment by ajithtunerlabs** [comment]
please don't load all the question address all the comments which I sent you through slack

File: controllers/v1/reportsController.js (hunk 2 above)
Comment (aman-tunerlabs) [reply controllers/v1/reportsController.js]: Resolved

File: module/solutions/helper.js (hunk 1 above)
Comment (aman-tunerlabs) [reply module/solutions/helper.js]: Resolved

**Pull request merged**